scripts/src/public-api-methods.ts

```diff
@@ -0,0 +1,249 @@
+    "conversations.list",
+    "conversations.listConnectInvites",
+    "conversations.mark",
+    "conversations.members",
+    "conversations.open",
+    "conversations.rename",
+    "conversations.replies",
+    "conversations.setPurpose",
+    "conversations.setTopic",
+    "conversations.unarchive",
+    "dialog.open",
+    "dnd.endDnd",
+    "dnd.endSnooze",
+    "dnd.info",
+    "dnd.setSnooze",
+    "dnd.teamInfo",
+    "emoji.list",
+    "enterprise.auth.idpconfig.apply",
+    "enterprise.auth.idpconfig.get",
+    "enterprise.auth.idpconfig.list",
+    "enterprise.auth.idpconfig.remove",
+    "enterprise.auth.idpconfig.set",
+    "files.comments.delete",
+    "files.delete",
+    "files.info",
+    "files.list",
+    "files.revokePublicURL",
+    "files.sharedPublicURL",
+    "files.upload",
+    "files.remote.add",
+    "files.remote.info",
+    "files.remote.list",
+    "files.remote.remove",
+    "files.remote.share",
+    "files.remote.update",
+    "migration.exchange",
+    "oauth.access",
+    "oauth.v2.access",
+    "oauth.v2.exchange",
+    "openid.connect.token",
+    "openid.connect.userInfo",
+    "pins.add",
+    "pins.list",
+    "pins.remove",
+    "reactions.add",
+    "reactions.get",
+    "reactions.list",
+    "reactions.remove",
+    "reminders.add",
+    "reminders.complete",
+    "reminders.delete",
+    "reminders.info",
+    "reminders.list",
+    "rtm.connect",
+    "rtm.start",
+    "search.all",
+    "search.files",
+    "search.messages",
+    "stars.add",
+    "stars.list",
+    "stars.remove",
+    "team.accessLogs",
+    "team.billableInfo",
+    "team.info",
+    "team.integrationLogs",
+    "team.billing.info",
+    "team.preferences.list",
+    "team.profile.get",
+    "tooling.tokens.rotate",
+    "usergroups.create",
+    "usergroups.disable",
+    "usergroups.enable",
+    "usergroups.list",
+    "usergroups.update",
+    "usergroups.users.list",
+    "usergroups.users.update",
+    "users.conversations",
+    "users.deletePhoto",
+    "users.getPresence",
+    "users.identity",
+    "users.info",
+    "users.list",
+    "users.lookupByEmail",
+    "users.setActive",
+    "users.setPhoto",
+    "users.setPresence",
+    "users.profile.get",
+    "users.profile.set",
+    "views.open",
+    "views.publish",
+    "views.push",
+    "views.update",
+    "workflows.stepCompleted",
+    "workflows.stepFailed",
+    "workflows.updateStep",
+    "channels.create",
+    "channels.info",
+    "channels.invite",
+    "channels.mark",
+    "groups.create",
+    "groups.info",
+    "groups.invite",
+    "groups.mark",
+    "groups.open",
+    "im.list",
+    "im.mark",
+    "im.open",
+    "mpim.list",
+    "mpim.mark",
+    "mpim.open",
+    // upcoming platform 2.0 methods we want available but aren't listed quite yet
+    "functions.completeError",
+    "functions.completeSuccess",
+    "apps.datastore.delete",
+    "apps.datastore.get",
+    "apps.datastore.put",
+    "apps.datastore.query",
```

Comment:
nit: Again, not sure how this is generated, but since these are from a different source I'm wondering if we want to keep these platform 2.0 methods in a separate array and concat the two arrays. That seems like the distinction would be a bit more obvious and intentional.